src/openrct2-ui/input/KeyboardShortcut.cpp

```diff
@@ -1020,6 +1021,12 @@ static void shortcut_decrease_element_height()
     }
 }
 
+static void shortcut_toggle_clearance_checks()
+{
+    auto setCheatAction = SetCheatAction(CheatType::DisableClearanceChecks, gCheatsDisableClearanceChecks == 0 ? 1 : 0);
```

Comment:
`gCheatsDisableClearanceChecks` is a bool, so this will do:
```suggestion
    auto setCheatAction = SetCheatAction(CheatType::DisableClearanceChecks, gCheatsDisableClearanceChecks ? 1 : 0);
```